src/algebraic_topology/simplex_category.lean

```diff
@@ -0,0 +1,252 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+
+import order.category.NonemptyFinLinOrd
+import category_theory.skeletal
+import data.finset.sort
+import tactic.linarith
+
+/-! # The simplex category
+
+We construct a skeletal model of the simplex category, with objects `ℕ` and the
+morphism `n ⟶ m` being the monotone maps from `fin (n+1)` to `fin (m+1)`.
+
+We show that this category is equivalent to `NonemptyFinLinOrd`.
+-/
+
+universe variables u
+
+open category_theory
+
+/-- The simplex category:
+* objects are natural numbers `n : ℕ`
+* morphisms from `n` to `m` are monotone functions `fin (n+1) → fin (m+1)`
+-/
+@[derive inhabited]
+def simplex_category := ℕ
+
+namespace simplex_category
+
+instance : small_category simplex_category :=
+{ hom := λ n m, preorder_hom (fin (n+1)) (fin (m+1)),
+  id := λ m, preorder_hom.id,
+  comp := λ _ _ _ f g, preorder_hom.comp g f, }
+
+@[simp] lemma id_apply {n : simplex_category} (i : fin (n+1)) :
+  (𝟙 n : fin _ → fin _) i = i := rfl
+@[simp] lemma comp_apply {l m n : simplex_category} (f : l ⟶ m) (g : m ⟶ n) (i : fin (l+1)) :
+  (f ≫ g) i = g (f i) := rfl
+
+section generators
+/-!
+## Generating maps for the simplex category
+
+PROJECT: prove that the simplex category is equivalent to
```

Comment:
PROJECT = TODO?

Reply:
I guess I have always used `PROJECT` to mean `TODO, but not in this PR`. I'll change to `TODO`.